src/main/java/com/cepengagementservice/Config/CorsConfig.java

```diff
@@ -5,21 +5,24 @@
 import org.springframework.web.servlet.config.annotation.CorsRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 
-    @Configuration
-    public class CorsConfig{
+/**
+ * Allows for CORS
+ * @author Unknown
+ *
+ */
```

Comment:
The class seems simple but more detail would be appreciated, inlcuding the method it has.